app/models/post.rb

```diff
@@ -97,6 +97,14 @@ def answer?
     post_type_id == Answer.post_type_id
   end
 
+  def has_pending_suggested_edit?
+    !pending_suggested_edit.nil?
+  end
+
+  def pending_suggested_edit
+    SuggestedEdit.where(post_id: self.id, active: true).take
```

Comment:
Use `.any?` instead of `.take` here - it'll use a more efficient ActiveRecord query.

Reply:
I think I can use it only for the [has_]pending_suggested_edit? method, though, right? (name change recommended by rubocop)

Reply:
Yeah, that's right. Misplaced this comment, by the looks. `.any?` to check for existence. For _this_ method, you want `.last` instead of `.take`, again because it's more efficient SQL.